search-processors/src/main/java/org/opensearch/searchpipelines/questionanswering/generative/llm/Llm.java

```diff
@@ -22,5 +22,10 @@
  */
 public interface Llm {
 
+    enum ModelProvider {
+        OPENAI,
+        BEDROCK
```

Comment:
Could we add a comment here? When I look at it I feel we are supporting all the llms in bedrock, which is not. So a comment is useful here I think?

Reply:
Got it!

Reply:
Also, created #1495

Reply:
Thanks!